zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -146,7 +148,11 @@ public synchronized void init() {
         logger.info("Create remote interpreter {}", getClassName());
         property.put("zeppelin.interpreter.localRepo", localRepoPath);
         client.createInterpreter(groupId, noteId,
-            getClassName(), (Map) property);
+          getClassName(), (Map) property);
+
+        // Push angular object loaded from JSON file to remote interpreter
+        pushAngularObjectRegistryToRemote(client);
```

Comment:
init() will be invoked multiple times if there're multiple interpreters in the same interpreter group.
I think it's better make it call only once after interpreter process created.